docs/application/dotnet/guides/user-interface/nui/grid-layout.md

```diff
@@ -28,6 +36,17 @@ The following figure shows how to set the `Colomns` and the `Rows` properties. S
 
 ![Column](./media/columnLayout.png)
 
+<div id="TabSection1">
```

Comment:
This tab section is not visible by default. It should shows first tab after opening this page.